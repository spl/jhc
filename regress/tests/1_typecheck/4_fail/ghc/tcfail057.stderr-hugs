ERROR "tcfail057.hs":5 - Undefined type constructor "RealFrac"
