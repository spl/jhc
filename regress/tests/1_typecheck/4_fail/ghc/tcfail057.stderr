
tcfail057.hs:5:7:
    Predicate `RealFrac a' used as a type
    In the type signature for `f': f :: (RealFrac a) -> a -> a
